docs/docs/cmd/tenant/id/id-get.md

```diff
@@ -32,3 +32,23 @@ Get Microsoft 365 tenant ID of the the tenant to which you are currently logged
 ```sh
 m365 tenant id get
 ```
+
+## Response
+
+=== "JSON"
+
+    ``` json
+    "e65b162c-6f87-4eb1-a24e-1b37d3504663"
+    ```
+
+=== "Text"
+
+    ``` text
+    e65b162c-6f87-4eb1-a24e-1b37d3504663
+    ```
+
+=== "CSV"
+
+    ``` CSV
```

Comment:
Could we remove the space after the code fence and write the language in small letters please? Just like we do in our example docs.
Could you apply this to all other "Response" code fences as well?

```suggestion
    ```csv
```